agent/app/agent.go

```diff
@@ -435,6 +435,17 @@ func (agent *ecsAgent) getEC2InstanceID() string {
 	return instanceID
 }
 
+// getoutpostARN gets the Outpost ARN from the metadata service
+func (agent *ecsAgent) getoutpostARN() string {
```

Comment:
synced offline. let's move this to `agent_unix.go` to make the IMDS call, since this is not supported in Windows.